website/migrations/0242_link_streak_badge_icons.py

```
@@ -0,0 +1,36 @@
+        "15 Day Streak": "badges/15-day-streak.PNG",
+        "Monthly Streak": "badges/30-day-streak.PNG",
+        "100 Day Streak": "badges/100-day-streak.PNG",
+        "Six Month Streak": "badges/180-day-streak.PNG",
+        "Yearly Streak": "badges/365-day-streak.PNG",
+    }
+    for title, icon_relative_path in badge_icon_map.items():
+        badge = Badge.objects.filter(title=title).first()
+        if badge:
+            static_icon_path = os.path.join("website", "static", "img", icon_relative_path)
+            if os.path.exists(static_icon_path):
+                media_icon_path = os.path.join(settings.MEDIA_ROOT, "badges", os.path.basename(static_icon_path))
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)